lib/game_stats.rb

```diff
@@ -1,4 +1,94 @@
 
 
+
+
 module GameStats
+
+	def highest_total_score(season) # integer
```

Comment:
Nice and clean. Good use of helper methods.

Reply:
I'm seeing that you're passing in seasons as an argument for your methods. I'm not sure if you maybe looked at my code as a model, but I believe only the Season Stats uses season as a parameter. In this Game Stats section I think we should be creating a method that looks at the entire set of data, not just a specific season